helm-chart/splunk-operator/values.yaml

```diff
@@ -0,0 +1,137 @@
+# Splunk image
+image:
+  repository: docker.io/splunk/splunk:edge
+
+# The kube-rbac-proxy is a small HTTP proxy for a single upstream, that can perform RBAC 
+# authorization against the Kubernetes API.
+# reference: https://github.com/brancz/kube-rbac-proxy
+kubeRbacProxy:
+
+  # Specify kube-rbac-proxy image
+  image: 
+    repository: gcr.io/kubebuilder/kube-rbac-proxy
+    pullPolicy: IfNotPresent
+    tag: "v0.11.0"
+
+  # Set resource requests and limits for kube-rbac-proxy container
+  resources:
+    limits:
+      cpu: 200m
+      memory: 100Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+
+# Splunk Operator configurations
+splunkOperator: 
+
+  # Enable to deploy Splunk Operator
+  enabled: true
+
+  # Splunk operator image and pull policy
+  # reference: https://github.com/splunk/splunk-operator
+  image:
+    repository: docker.io/splunk/splunk-operator:latest
+    pullPolicy: IfNotPresent
```

Comment:
nit: `IfNotPresent` is already by default.

Reply:
Need to include `pullPolicy` key if the user decides they want to change `IfNotPresent`, default is also listed explicitly for the user.